lib/rubocop/cop/lint/float_comparison.rb

```diff
@@ -32,12 +40,20 @@ module Lint
       #   # good - comparing against nil
       #   Float(x, exception: false) == nil
       #
+      #   # good - using epsilon comparison in case statement
+      #   if (value - 1.0).abs < Float::EPSILON
+      #     foo
+      #   elsif (value - 2.0).abs < Float::EPSILON
+      #     bar
+      #   end
+      #
       #   # Or some other epsilon based type of comparison:
       #   # https://www.embeddeduse.com/2019/08/26/qt-compare-two-floats/
       #
       class FloatComparison < Base
         MSG_EQUALITY = 'Avoid equality comparisons of floats as they are unreliable.'
         MSG_INEQUALITY = 'Avoid inequality comparisons of floats as they are unreliable.'
+        MSG_CASE = 'Avoid using float literals in `when` clauses. Use epsilon comparison instead.'
```

Comment:
I'm guessing the part about `epsilon` is relevant in all cases, but I'd rather have the 3 types of messages be consistent in their wording.